backend/test_nightly/test_execution_minutes_quota.py

```diff
@@ -105,6 +105,52 @@ def test_set_execution_mins_extra_quotas(org_with_quotas, admin_auth_headers):
         assert update["update"]
 
 
+def test_add_execution_mins_extra_quotas(
+    org_with_quotas, admin_auth_headers, preshared_secret_auth_headers
+):
+    r = requests.post(
+        f"{API_PREFIX}/orgs/{org_with_quotas}/quotas/add",
```

Comment:
I think I removed the `/quotas/add` endpoint, as we weren't using it directly anymore (using /subscription/addMinutes instead), but could still test this with `/quotas/update`